### DISCUSSION TITLE
Multiple request body type for same endpoint

### DISCUSSION THREAD

**Discussion by Sharathmk99** [fastapi/fastapi#7537] [Questions]
### First check

* [YES ] I used the GitHub search to find a similar issue and didn't find it.
* [ YES] I searched the FastAPI documentation, with the integrated search.
* [ YES] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I add multiple request body for same endpoint. I wanted to accept application/json and application/pdf for same endpoint. Lets say i have exposed http://localhost:5000/test which should accept application/json as well as application/pdf. I wanted to write business logic based on request body type.?

Is it possible to expose only one endpoint with multiple request body type like application/json and application/pdf?

**Comment by phy25** [comment]
Duplicate of #990.

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :bow: 

Let's go to the original issue. I'll close this one.
